frontend/src/components/Course/CourseInfo/CourseInfo.tsx

```diff
@@ -12,7 +12,13 @@ import {
   LessonsIcon,
 } from './styled';
 
-const CourseInfo: React.FC<ICourseInfo> = ({ duration, lessons, type }) => (
+interface Props {
+  duration: string | undefined;
+  lessons: number | undefined;
```

Comment:
```suggestion
  duration?: string;
  lessons?: number;
```

I understand, that in ts its different cases, but for us, its good to write like this